src/components/ReportDocumentPDF/ReportDocumentPDF.jsx

```diff
@@ -73,7 +73,7 @@ const styles = StyleSheet.create({
   },
 });
 
-const ReportDocumentPDF = ({ selectedReport }) => {
+const ReportDocumentPDF = ({ selectedReport, sels }) => {
```

Comment:
This component gets re-used in two other places: by the "Download" button and the `StandardCardTable` component. It's safe to modify this component props because the former will be removed and the latter never rendered this component:
```
// No parent component overrides this
const { enablepdfexport = false } = { ...props };

// [...]

{enablepdfexport && (
  <Button
    as={PDFDownloadLink}
    document={<ReportDocumentPDF selectedReportCards={cards} />}
    fileName="report.pdf"
    boxShadow="base"
  >
    Download PDF
  </Button>
)}
```